components/match2/wikis/wildrift/match_group_input_custom.lua

```diff
@@ -449,11 +450,13 @@ end
 function mapFunctions.getParticipants(map, opponents)
 	local participants = {}
 	local championData = {}
+	local getCharacterName = FnUtil.curry(MatchGroupInput.getCharacterName, ChampionNames)
 	for opponentIndex = 1, MAX_NUM_OPPONENTS do
 		for playerIndex = 1, MAX_NUM_PLAYERS do
 			local champ = map['t' .. opponentIndex .. 'c' .. playerIndex]
-			championData['team' .. opponentIndex .. 'champion' .. playerIndex] =
-				ChampionNames[champ] or champ
+			championData['team' .. opponentIndex .. 'champion' .. playerIndex] = getCharacterName(champ)
+			local ban = map['t' .. opponentIndex .. 'b' .. playerIndex]
```

Comment:
i have no idea about wildrift
is always number of bans == number of players per team?

Reply:
Maybe it is a good idea to not mess with the logic. They have in some cases less bans than players ...